docs/client/jdbc/hive_jdbc.md

```diff
@@ -76,7 +84,3 @@ jdbc:hive2://<host>:<port>/<dbName>;<sessionVars>?<kyuubiConfs>#<[spark|hive]Var
 jdbc:hive2://localhost:10009/default;hive.server2.proxy.user=proxy_user?kyuubi.engine.share.level=CONNECTION;spark.ui.enabled=false#var_x=y
 ```
 
-## Unsupported Hive Features
```

Comment:
Can we add a meaningful doc here?

Reply:
Emm, it's actually supports thrift HTTP mode

Reply:
I mean doc how to use http

Reply:
Already in `Connection URL over HTTP` part